lightning/src/routing/router.rs

```diff
@@ -237,6 +245,13 @@ impl Payee {
 	pub fn with_expiry_time(self, expiry_time: u64) -> Self {
 		Self { expiry_time: Some(expiry_time), ..self }
 	}
+
+	/// Includes a limit for the total CLTV expiry delta which is considered during routing
+	///
+	/// (C-not exported) since bindings don't support move semantics
+	pub fn with_max_total_cltv_expiry_delta(self, max_total_cltv_expiry_delta: u32) -> Self {
+		Self { max_total_cltv_expiry_delta: max_total_cltv_expiry_delta, ..self }
```

Comment:
nit: don't need to repeat the field name now it is not an option.

```rust
Self { max_total_cltv_expiry_delta, ..self }
```

Reply:
Fixed in https://github.com/lightningdevkit/rust-lightning/pull/1234/commits/8ea27e5faadead192baf0df9035a556ffcdd43a3